targets/TARGET_ARM_FM/TARGET_FVP_MPS2/PeripheralNames.h

```diff
@@ -52,6 +52,7 @@ typedef enum {
     ADC0_11
 } ADCName;
 
+#define SPI_COUNT 5
```

Comment:
For testing purposes it is required to define which interfaces should be used for testing:
```
#define SPI_TEST_MASTER SPI_2
#define SPI_TEST_SLAVE SPI_0
#define SPI_TEST_MASTER_PIN(SPI_PIN) SPI_2##_##SPI_PIN
#define SPI_TEST_SLAVE_PIN(SPI_PIN)  SPI_0##_##SPI_PIN
```

Please check https://github.com/ARMmbed/mbed-os/pull/7976#issuecomment-452295708.

Reply:
Can you try to run this test on Fast Model target? PR https://github.com/ARMmbed/mbed-os/pull/7976